django/core/mail/message.py

```diff
@@ -100,22 +100,62 @@ def forbid_multi_line_headers(name, val, encoding):
     return str(name), val
 
 
+def split_addr(addr, encoding):
+    """
+    Splits an address into local part and domain, properly encoded.
```

Comment:
Noted, will define properly encoded as 'MIME-word encoded` for the local part and `IDN-encoded` for the domain part. Do you want a reference to wiki/spec for what MIME-word encoded is in the docstring, or is the term enough?